controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/jobs.json

```diff
@@ -335,7 +335,7 @@
       "runs": [
         {
           "id": 1,
-          "url": "http://localhost:8080/application/v4/tenant/tenant1/application/application1/instance/instance1/job/run/run 1 of dev-us-east-1 for tenant1.application1.instance1",
+          "url": "http://localhost:8080/application/v4/tenant/tenant1/application/application1/instance/instance1/job?/run/run 1 of dev-us-east-1 for tenant1.application1.instance1",
```

Comment:
?

Reply:
Oops.

Reply:
The `?` is not the worst part — read the rest of the line!